app/initializers/ember-intl.js

```diff
@@ -55,8 +56,29 @@ export var registerIntl = function (container) {
 export default {
     name: 'ember-intl',
 
-    initialize: function (container, app) {
-        registerIntl(container);
-        app.intl = container.lookup('service:intl');
+    initialize(registry, application) {
```

Comment:
If in < 1.12.0 we should import and invoke `initializeIntl` from the instance-initializer

/cc @al3x-edge